extra-x11/nvidia-prime/spec

```diff
@@ -0,0 +1,2 @@
+VER=1.0
+DUMMYSRC=1
```

Comment:
Don't use `DUMMYSRC=1` unless you absolutely have to, and in this case, you don't.